docs/csharp/tutorials/intro-to-csharp/branches-and-loops.yml

```diff
@@ -270,13 +270,11 @@ items:
 
 - title: Congratulations!
   content: |
-    You've completed the "branches and loops" tutorial. 
-    
-    You can continue these tutorials on your own development environment. Learn the basics of [local development](local-environment.md) and then pick a tutorial. You can try this same exercise, move directly to the next tutorial, or start again at the beginning of the series with the [numbers in C#](numbers-in-csharp-local.md) tutorial.
+    You've completed the "branches and loops" interactive tutorial. You can click the **Interpolated strings** link below to start the next interactive tutorial, or you can visit the [.NET site](https://www.microsoft.com/net/learn/dotnet/hello-world-tutorial) to download the .NET Core SDK, create a project on your machine, and keep coding. Once you've completed those steps, the .NET site has a link labeled **Keep Learning** that will send you back to these tutorials so you can continue working. 
```

Comment:
The link isn't labeled "Keep learning"; the section is labeled "Keep Learning". The link reads, "Quick start: Collections" I like the phrasing that you used for interpolated strings and later: "The "Keep Learning" step brings you back to these tutorials."